ffc/codegeneration/jit.py

```diff
@@ -214,86 +217,162 @@
 """
 
 
+def get_cached_module(module_name, object_names, parameters):
+    cache_dir = parameters.get("cache_dir",
+                               "compile_cache")
+
+    timeout = int(parameters.get("timeout", 100))
+
+    c_filename = cache_dir + "/" + module_name + ".c"
```

Comment:
Use `pathlib` (https://docs.python.org/3/library/pathlib.html#module-pathlib) to manipulate paths and filenames.